### PR TITLE
Change componentWillReceiveProps to getDerivedStateFromProps

### PR DESCRIPTION
**Pull request by Karl-EdwardFPJeanMehu**
Fix for issue #97 UNSAFE_componentWillReceiveProps

### CODE REVIEW

**Comment by petrsvihlik** [comment]
Thanks @Karl-EdwardFPJeanMehu , we'll take a look at it soon!

File: src/Components/ContactMap.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class MapContainer extends Component {
   }
 
   //TODO: Method will be removed in React 17, will need to be rewritten if still required.
```
Comment (Simply007) [inline src/Components/ContactMap.js]: Clould you please remove this comment line.

**Comment by Simply007** [comment]
@Karl-EdwardFPJeanMehu - if you need any help or advice, just feel free to ask!

**Comment by Simply007** [comment]
Hello @Karl-EdwardFPJeanMehu,

due to inactivity, I have decided to merge your pull request and revert changes in ContactMap component right after the merge. I will close the issue and create a new one for refactoring ContactMap component.

Thanks for your contribution!

**Comment by Simply007** [comment]
Hey @Karl-EdwardFPJeanMehu,

we would like to thank you for contributing to Kentico's open-source project. As a sign of our appreciation, we would like to send you some [Kentico Developer Community SWAG](https://twitter.com/PetrSvihlik/status/1059776942537392129)!

Please fill out your [postal address](https://goo.gl/forms/gRnURSDlXMX4ctRZ2) if you are interested.

_This comment was automatically generated. If you've submitted more than one pull request, it's OK to fill out the form just once._

Kentico Developer Community

_If there’s anything we can do to help, please don’t hesitate to reach out to us at developerscommunity@kentico.com_

**Pull request merged**